scipy/stats/__init__.py

```diff
@@ -334,7 +334,7 @@
 
 
 Quasi-Monte Carlo
-=================
```

Comment:
I think you're going for a separation between "**Probability** distributions" and "**Statistical** Functions", but I'm not sure things are as neat as that. QMC is closer to probability distributions than statistical functions, no? In any case, QMC and Masked Statistics Functions are their own namespaces, so I think they should stay higher-level headings.

Reply:
I undid all the changes to the headings and heading markup.